mp/src/game/shared/neo/weapons/weapon_neobasecombatweapon.cpp

```diff
@@ -94,6 +103,7 @@ CNEOBaseCombatWeapon::CNEOBaseCombatWeapon( void )
 
 void CNEOBaseCombatWeapon::Spawn()
 {
+	PrecacheParticleSystem("ntr_muzzle_source");
 	// If this fires, either the enum bit mask has overflowed,
 	// this derived gun has no valid NeoBitFlags set,
 	// or we are spawning an instance of this base class for some reason.
```

Comment:
This assertion is also outdated for the same reasons as the previous one, and is triggering incorrectly. Should rewrite as `Assert(GetNeoWepBits() != NEO_WEP_INVALID)`.